From c041459ad447e16f97f56391dce87a959c655cb8 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Mon, 11 Jul 2016 17:21:56 -0700
Subject: [PATCH 1760/2848] HADOOP-13297. Add missing dependency in setting
 maven-remote-resource-plugin to fix builds.
 Contributed by Sean Busbey.

(cherry picked from commit 7bd5d4272cd686e06c5d5fcc489b69312dacb47b)
(cherry picked from commit 02b037f6257e7379b6bbd656ccbe4472d3f46c40)
(cherry picked from commit 59818313d490e8e509b9cc11cce4f5aec30c4a93)

Change-Id: I143fd8ec06d4c6e3203a6a261f122e9804449cb2
---
 hadoop-project/pom.xml |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index be74d97..3b189b7 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -1108,6 +1108,13 @@
             <resourceBundle>org.apache.hadoop:hadoop-build-tools:${project.version}</resourceBundle>
           </resourceBundles>
         </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-build-tools</artifactId>
+            <version>${project.version}</version>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
             <goals>
-- 
1.7.9.5

